EasyLearn/app/assets/stylesheets/styles.css

```diff
@@ -0,0 +1,145 @@
+.contenido{
+  background-color: #282828;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.nav{
+  align-items: center;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 80%;
+}
+ 
+.nav__items{
+  align-items: center;
+  display: flex;
+  font-size: 25px;
+  gap: 2rem;
+}
+ 
+.dropdown > .btn{
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.main__content{
+  border-top: 1px solid orange;
+  background-color: #282828;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.4rem;
+  height: 99.3vh;
+  width: 100%;
+}
+
+.aside__profile{
+  display: flex;
+  align-items: center;
+  padding-top: 4rem;
+  padding-left: 1rem;
+  justify-content: center;
+}
+.aside__picture{
+  width: 35%;
+  
+}
+.aside__img{
+  border-radius: 100px;
+}
+
+.aside__helpers{
+  margin: 0 auto;
+  width: 59%;
+  padding-top: 6rem;
+}
+
+.aside__helpers-buttons{
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  transition: all .1s ease-in-out;
+}
+
+.aside__helpers-buttons:hover{
+  color: #858585;
```

Comment:
For color can be use variables, can found some documentation [here](https://developer.mozilla.org/es/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
Done.